internals/daemon/daemon.go

```diff
@@ -864,6 +871,10 @@ func New(opts *Options) (*Daemon, error) {
 		httpAddress:         opts.HTTPAddress,
 	}
 
+	// Superuser and process owner can do anything.
+	d.admins = []sys.UserID{0, sysGetuid()}
```

Comment:
I know two 0's in this list doesn't hurt if it's running as root, but it's also a bit strange. How about:

```suggestion
	d.admins = []sys.UserID{0}
	if uid := sysGetuid(); uid != 0 {
		d.admins = append(d.admins, uid)
	}
```